src/js/components/action.jsx

```diff
@@ -112,6 +117,15 @@ export class ActionComponent extends Component {
                     </div>
                     );
             }
+        } else if (this.props.type === 'postback') {
+            return (
+                <div className='sk-action'>
+                    <button className='btn btn-sk-primary'
+                            onClick={ this.onPostbackClick.bind(this) }>
```

Comment:
I might be better to inline that function since it's not doing much:
`onClick={() => postPostback(this.props._id)}`